rad/src/script.rs

```diff
@@ -252,7 +252,11 @@ pub fn create_radon_script_from_filters_and_reducer(
     )
     .map_err(|_| unknown_reducer(i128::from(reducer)))?;
     match rad_reducer {
-        RadonReducers::AverageMean | RadonReducers::Mode | RadonReducers::AverageMedian => {}
+        RadonReducers::AverageMean
+        | RadonReducers::Mode
+        | RadonReducers::AverageMedian
+        | RadonReducers::HashConcatenate
+        | RadonReducers::Unwrap => {}
```

Comment:
This function needs a context, the new operators can only be considered valid if the WIP is active.

Reply:
It is fixed in #2063, because it also affect to the median